tests/jballerina-unit-test/src/test/resources/test-src/typedefs/tuple-type-definitions-cyclic.bal

```diff
@@ -327,6 +327,26 @@ function testUnionWithCyclicTuplesHashCode() {
     "\"superType\":[\"|\", [\"|\", \"int\", \"float\"], [\"tuple\", \"never\", [\"|\", \"int\", \"string\"]]]}"); 
 }
 
+type Q1 [Q1];
```

Comment:
Should we add test cases when a type is a tuple of a different cyclic type? i.e. `Q7 = [Q1]`